tasks/users.yml

```diff
@@ -13,7 +13,7 @@
     port: "{{postgresql_port}}"
     state: present
     login_user: "{{postgresql_admin_user}}"
-  no_log: true
+#  no_log: true
```

Comment:
Rather than commenting it out, should we either set it to false, or remove the line entirely?

Reply:
Reenabled. no_log: true suppresses the output of parameters used to  create the users. You may consider it a security feature.